usability_task_force

```diff
@@ -0,0 +1,9 @@
+# Usability Task Force
+Although there is a concerted effort from the UE team to solicit feedback on the existing features of the KBase platform,
+there is no central resource to coordinate, for developers, usability tests of prototypes or mockups of proposed new
+user-facing features. The result is that most developers do not do any usability testing until after most of the design
+and coding work is done, at which point it is much more expensive to make significant changes. This proposal will create
+an ongoing "Usability Task Force" that will maintain a set of resources for all project developers on how and when
```

Comment:
Even though it is not the responsibility of a PD to pick the team (necessarily), I think the answer is "no". This is primarily intended as a fine-grained level of design testing for guiding the implementation, not for choosing which kinds of functionality the platform should provide.